.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,23 +1,24 @@
 ---
 name: 🐛 Bug Report
-about: Report a reproducible bug in the current release of DiffSync
+about: Report a reproducible bug in the current release of diffsync
```

Comment:
Agreed. We need to address this at [the source](https://github.com/networktocode-llc/cookiecutter-ntc/blob/main/python/%7B%7Bcookiecutter.project_slug%7D%7D/.github/ISSUE_TEMPLATE/bug_report.md), because currently this is cruft working as intended.